src/promptflow/promptflow/azure/operations/_flow_operations.py

```diff
@@ -132,6 +144,37 @@ def create_or_update(self, flow: Union[str, Path], display_name=None, type=None,
 
         return result_flow
 
+    def _update_azure_flow(self, flow: str, display_name, **kwargs):
+        """Update an existing flow in azure."""
+        body = {
+            "flow_name": display_name,
+            "description": kwargs.get("description", None),
+            "tags": kwargs.get("tags", None),
+        }
+        body = {k: v for k, v in body.items() if v is not None}
+        logger.debug(f"Updating flow {flow!r} with data {body}.")
+
+        try:
+            self._service_caller.update_flow(
+                subscription_id=self._operation_scope.subscription_id,
+                resource_group_name=self._operation_scope.resource_group_name,
+                workspace_name=self._operation_scope.workspace_name,
+                flow_id=flow,
+                body=body,
+            )
+        except Exception as e:
+            raise FlowOperationError(
+                f"Failed to update azure flow {flow!r} due to: {str(e)}. If the flow is not found in azure, "
+                f"please make sure the flow name is correct. If you are trying to create a new flow "
+                f"from local flow folder, please make sure the flow folder exists and try again."
+            ) from e
+
+        updated_flow = self.get(flow)
+        flow_dict = updated_flow._to_dict()
+        print(f"Flow updated successfully:\n{json.dumps(flow_dict, indent=4)}")
```

Comment:
is intensionally using print not log here?

Reply:
Updated to log info.